src/Microsoft.Crank.AzureDevOpsWorker/RetryHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Crank.AzureDevOpsWorker
+{
+    internal class RetryHandler : DelegatingHandler
```

Comment:
Can/should we use https://github.com/dotnet/extensions/tree/main/src/Libraries/Microsoft.Extensions.Http.Resilience?

Reply:
🤟